services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -0,0 +1,116 @@
+// components
+import { Button, Image, Td, Tr } from "@chakra-ui/react";
+// utils
+import { AnyObject, EntityShape } from "types";
+// assets
+import deleteIcon from "assets/icons/icon_cancel_x_circle.png";
+import unfinishedIcon from "assets/icons/icon_error_circle.png";
+
+export const EntityRow = ({
+  entity,
+  verbiage,
+  openAddEditEntityModal,
+  openDeleteEntityModal,
+}: Props) => {
+  const { programName, planName } = entity;
+
+  const reportingPeriod = `${entity.reportingPeriodStartDate} to ${entity.reportingPeriodEndDate}`;
+  const eligibilityGroup = () => {
+    if (entity["eligibilityGroup-otherText"]) {
+      return entity["eligibilityGroup-otherText"];
+    }
+    return entity.eligibilityGroup[0].value;
```

Comment:
Can this ever not have a 0 index?

Reply:
why is it always the first index if it isn't other-Text?

Reply:
just trying to understand this code and I'm failing, no notes on how to change it